src/components/common/SpaceTree.js

```diff
@@ -0,0 +1,222 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import SvgIcon from '@material-ui/core/SvgIcon';
+import { withTranslation } from 'react-i18next';
+import { fade, withStyles } from '@material-ui/core/styles';
+import TreeView from '@material-ui/lab/TreeView';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+import TreeItem from '@material-ui/lab/TreeItem';
+import { connect } from 'react-redux';
+import { getSpaceTree, patchAppInstance } from '../../actions';
+import {
+  TREE_VIEW_MAX_HEIGHT,
+  TREE_VIEW_MAX_WIDTH,
+} from '../../config/settings';
+
+const MinusSquare = props => {
```

Comment:
For this repo this is OK, but in general it's better to put each SVG under its own component inside an `svgs` folder.